### ISSUE TITLE
Would be good to start and stop stream on Record/Stop not to leave mic on on the page

### ISSUE THREAD

**Issue by d668** [sb2702/audioRecord.js#3]
I moved the audioRecorder.requestDevice into startRecording function but how to stop the stream?
thanks

**Comment by sb2702** [comment]
In the original code - the audioRecord doesn't actually do the recording - audioRecorder.requestDevice provides the 'recorder' object, which has recorder.record()  and recorder.stop() methods.

Edit: I'm re-reading this - and I see what you're saying. the stop and start methods don't actually stop the stream - let me look into this

**Comment by stefan-reich** [comment]
Yeah, the mic icon should really disappear... it's still blinking after recording stopped.

Nice tool BTW, I'll use it.
